www/auto-font/index.html

```diff
@@ -13,494 +13,482 @@
     <li><a href="/settingscommands/">Settings&nbsp;&amp;&nbsp;Commands</a></li>
     <li><a href="/staff/">Staff&nbsp;&amp;&nbsp;Contact</a></li>
     <li><a href="/funding/">Funding</a></li>
-  </ul>
+</ul>
 head: |
-    <style>
-        .autofont * {
+<style>
```

Comment:
and here. These blocks with | have to be indented.

Reply:
😄 got it